packages/nois-protocol/src/ibc_msg.rs

```diff
@@ -10,15 +10,34 @@ pub enum InPacket {
     RequestBeacon {
         /// Beacon publish time must be > `after`
         after: Timestamp,
+        /// The callback data set by the proxy in a proxy specific format.
+        callback: Binary,
```

Comment:
We can easily keep this enum case compatible by avoiding the rename origin -> callback. Then we only have an `RequestScheduleJob` addition here.